### ISSUE TITLE
ConfigGroup does not load urls

### ISSUE THREAD

**Issue by adaniline-traderev** [pulumi/pulumi-kubernetes#448]
This should be implemented, according to #150, but when I try to load several yamls by url with ConfigGroup, it does not work. ConfigFile works:
```
new k8s.yaml.ConfigGroup("jenkins-operator", {
    files: [
        'https://raw.githubusercontent.com/VirtusLab/jenkins-operator/master/deploy/crds/virtuslab_v1alpha1_jenkins_crd.yaml',
        'https://raw.githubusercontent.com/VirtusLab/jenkins-operator/master/deploy/service_account.yaml',
        'https://raw.githubusercontent.com/VirtusLab/jenkins-operator/master/deploy/role.yaml',
        'https://raw.githubusercontent.com/VirtusLab/jenkins-operator/master/deploy/role_binding.yaml',
        'https://raw.githubusercontent.com/VirtusLab/jenkins-operator/master/deploy/operator.yaml',
    ],
    transformations: [
        (obj => obj.metadata.namespace = namespace)
    ]
}, {
    providers: {
        kubernetes: k8sProvider
    }
});
```

**Comment by lblackstone** [comment]
I took a look at this, and it seems that the URL support wasn't actually implemented for `ConfigGroup` in #150. Until this gets fixed, you can still load them with `ConfigFile`.

**Comment by hausdorff** [comment]
I'm skeptical this _needs_ to go into Q3, but it's also small.
